authorize.py

```diff
@@ -70,19 +75,35 @@ def authorize_then_consume(event, context):
             rv['statusCode'] = 400
     return rv
 
-def create_user(event, context, mlh = False):
+@ensure_schema({
+    "type": "object",
+    "properties": {
+        "email": {"type": "string", "format": "email"},
+        "password": {"type": "string"},
+        "link": {"type": "string"},
+        "github": {"type": "string"},
+        "major": {"type": "string"},
+        "short_answer": {"type": "string"},
+        "shirt_size": {"type": "string"},
+        "first_name": {"type": "string"},
+        "last_name": {"type": "string"},
+        "dietary_restrictions": {"type": "string"},
+        "special_needs": {"type": "string"},
+        "date_of_birth": {"type": "string"},
+        "school": {"type": "string"},
+        "grad_year": {"type": "string"},
+        "gender": {"type": "string"},
+        "level_of_study": {"type": "string"},
+        "ethnicity": {"type": "string"},
+        "phone_number": {"type": "string"}
```

Comment:
Might be worth making this an int

Reply:
Will make an issue so that frontend can catch up after this hackru